generators/go-v2/sdk/src/endpoint/utils/getPaginationInfo.ts

```diff
@@ -432,7 +432,7 @@ function getResponsePropertySetter({
             writer.writeNode(
                 getPropertyNilCheckCondition({
                     variableName: "response",
-                    propertyPath: responsePropertyPath
+                    propertyPath: responsePropertyPath.map((item) => item.name)
```

Comment:
hrm is this sufficient? Or do we need to unpack the name and then use the originalName or safeName or whatever